crates/wac-graph/tests/graphs/implicit-resource-import/encoded.wat

```diff
@@ -0,0 +1,57 @@
+(component
+  (type (;0;)
+    (instance
+      (export (;0;) "x" (type (sub resource)))
+    )
+  )
+  (import "foo:dependency/types" (instance (;0;) (type 0)))
+  (alias export 0 "x" (type (;1;)))
+  (import "x" (type (;2;) (eq 1)))
```

Comment:
I'm not sure if this is correct...

Reply:
This is correct; a `use` of a resource in a world is an import of that resource, not an import of the interface and then aliasing the export of that interface.